.gitignore

```diff
@@ -61,3 +61,10 @@
 .idea
 .vscode/ipch
 /newIDE/app/src/UI/Theme/**/*ThemeVariables.*
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
```

Comment:
For which OS is this?

Reply:
.Spotlight-V100
.Trashes
Are generated by apple macintosh, 
._* would be removed also ehthumbs.db